src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { FormEventHandler, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getNewTodoId, getUserById } from './utils';
+
+export const preparedTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todosList, setTodosList] = useState<Todo[]>(preparedTodos);
+
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number>(0);
```

Comment:
FYI:  TypeScript can often infer the types automatically based on the initial values. But you don't need to fix it, your syntax is also correct. It's up to you
```suggestion
  const [title, setTitle] = useState('');
  const [userId, setUserId] = useState(0);
```